### PR TITLE
#51 edit api requests

### PR DESCRIPTION
**Pull request by Alkhimovmv**
I changed the requests to match the backend. 
There is a hardcode here. When the database appears, I will remove it

### CODE REVIEW

File: frontend/src/components/GoPractice/GoPractice.tsx [hunk 1]
```diff
@@ -35,14 +35,35 @@ const GoPractice = (): JSX.Element => {
...
             level: level
         }
-        api.post(`/configure?first_language=${dataConfig.first_language}&second_language=${dataConfig.second_language}&level=${dataConfig.level}`, JSON.stringify(dataConfig))
-            .then(response => {
-                window.localStorage.setItem('dataUuid', JSON.stringify(response.data))
+        window.localStorage.setItem('firstLanguage', firstLanguage)
+        window.localStorage.setItem('secondLanguage', secondLanguage)
+        window.localStorage.setItem('level', level)
+        // const session_token = window.localStorage.getItem('session_token')
+        const session_token = '40b39433-f0a5-4d89-abf2-7649fdcf2e7b'
```
Comment (AlexKay28) [inline frontend/src/components/GoPractice/GoPractice.tsx:60]: is this ok to store session_token in code? maybe you will move it out of code as temporal config for testing now... at least it definitely should be removed later

Comment (AlexKay28) [reply frontend/src/components/GoPractice/GoPractice.tsx:60]: oh, i guess the commented code above is something what was replaced by this one) am I right?

File: frontend/src/components/Question/Question.jsx [hunk 2]
```diff
@@ -20,32 +20,48 @@ class Question extends React.Component {
...
-                const uuid = data.uuid;
-                const quid = res.data.quid;
-                this.setState({ question, uuid, quid});
-        })
     }
 
-    handleAnswerSubmit() {
+    handleQuestionSubmit() {
+        // const session_token = window.localStorage.getItem('session_token')
+        const session_token = '40b39433-f0a5-4d89-abf2-7649fdcf2e7b'
```
Comment (AlexKay28) [inline frontend/src/components/Question/Question.jsx:31]: yeah, i see it again... maybe its a good practice to put comments in code whose let others developers to know what's going on here

File: frontend/src/components/Question/Question.jsx [hunk 3]
```diff
@@ -20,32 +20,48 @@ class Question extends React.Component {
...
         const data = {
-            uuid: this.state.uuid,
-            quid: this.state.quid,
-            answer_user: this.state.answer_user
+            quid: window.localStorage.getItem('quid'),
+            user_answer: this.state.user_answer
         }
-        api.post(`/answer?uuid=${data.uuid}&quid=${data.quid}&second_language_phrase_answer=${data.answer_user}`, JSON.stringify(data))
+        // const session_token = window.localStorage.getItem('session_token')
+        const session_token = '40b39433-f0a5-4d89-abf2-7649fdcf2e7b'
```
Comment (AlexKay28) [inline frontend/src/components/Question/Question.jsx:55]: the same as above

**Comment by AlexKay28** [comment]
@Alkhimovmv

**Review by AlexKay28 — CHANGES_REQUESTED** [review]
answer asked questions in comments and add some changes in code (if its needed)

**Comment by Alkhimovmv** [comment]
> @Alkhimovmv if the interface has changed somehow, could you please attach screenshots in the comments please...

Yes, I moved the GoPractice component to the top and pinned it like we discussed
Снимок экрана от 2021-11-06 13-03-48
Снимок экрана от 2021-11-06 13-04-06
Снимок экрана от 2021-11-06 13-05-33

File: frontend/src/components/GoPractice/GoPractice.tsx (hunk 1 above)
Comment (Alkhimovmv) [reply frontend/src/components/GoPractice/GoPractice.tsx:60]: Yes, you're right, the commented code above is something what was replaced by this one. But I see no reason to move the session_token to the config, if with the appearance of database I remove this line and uncomment the code

File: frontend/src/components/Question/Question.jsx (hunk 2 above)
Comment (Alkhimovmv) [reply frontend/src/components/Question/Question.jsx:31]: Yes, it's better for understanding what's going on in this code.

**Comment by AlexKay28** [comment]
> > @Alkhimovmv if the interface has changed somehow, could you please attach screenshots in the comments please...
> 
> Yes, I moved the GoPractice component to the top and pinned it like we discussed Снимок экрана от 2021-11-06 13-03-48 Снимок экрана от 2021-11-06 13-04-06 Снимок экрана от 2021-11-06 13-05-33

You good, I see!) nice answer

**Comment by AlexKay28** [comment]
ok, watch for #39

**Review by alantishin — APPROVED** [review]

**Review by AlexKay28 — APPROVED** [review]

**Comment by Alkhimovmv** [comment]
[Handle new API](https://trello.com/c/vOSHBsKV/3-handle-new-api)

**Pull request merged**